templates/artifactory-oss/0/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '2'
+
+services:
+
+  artifactory:
+    image: docker.bintray.io/jfrog/artifactory-oss:5.1.3
+    ports:
+     - "8081"
```

Comment:
If you want to use lb instead expose port directly (recommended way), remove ports from artifactory, and add lb instance. Also, if you want the user could choose, you could add a variable in racnher-compose.yml and use go templating in docker-compose.yml.tpl 

As example, take a look at https://github.com/rancher/community-catalog/blob/master/templates/gocd/0/docker-compose.yml.tpl